src/Repository/AdherentRepository.php

```diff
@@ -550,4 +550,25 @@ public function countSupervisorsByGenderForReferent(Adherent $referent): array
             ->getArrayResult()
         ;
     }
+
+    public function countMembersByGenderForReferent(Adherent $referent): array
+    {
+        if (!$referent->isReferent()) {
+            throw new \InvalidArgumentException('Adherent must be a referent.');
+        }
+
+        return $this->createQueryBuilder('adherent', 'adherent.gender')
+            ->select('adherent.gender, COUNT(DISTINCT adherent) AS count')
```

Comment:
For `COUNT(DISTINCT adherent)` do we need to specify `adherent.id` instead or it's already the end result of the SQL ?

Reply:
Yes, it's already the end result of the SQL.